toolkit/templates/forms/pricing.html

```diff
@@ -30,6 +32,10 @@
     <span class="validation-message">
       {{ error }}
     </span>
+    {% elif message %}
+    <span class="message-message">
```

Comment:
I was trying to follow the pattern of `validation-wrapper` and `validation-message`.  But yeah, I knew at the time this is weird.